admin/metabox/class-metabox.php

```diff
@@ -571,6 +571,13 @@ public function do_meta_box( $meta_field_def, $key = '' ) {
 
 		switch ( $meta_field_def['type'] ) {
 			case 'pageanalysis':
+				$content_analysis_active = $this->options['content_analysis_active'];
+				$keyword_analysis_active = $this->options['keyword_analysis_active'];
+
+				if( $content_analysis_active === false && $keyword_analysis_active === false ) {
```

Comment:
Can you add a space after the `if`:
`if ( $content_analysis_active === false && $keyword_analysis_active === false ) {`